src/common/libschedutil/ops.c

```diff
@@ -23,29 +23,38 @@
 static void alloc_continuation (flux_future_t *f, void *arg)
 {
     schedutil_t *util = arg;
-    flux_t *h = util->h;
-    const flux_msg_t *msg = flux_future_aux_get (f, "schedutil::msg");
 
-    if (util == NULL) {
-        errno = EINVAL;
-        goto error;
+    if (!f || !util) {
+        flux_log (util->h, LOG_ERR, "alloc_continuation: invalid args");
+        return;
     }
-    const char *jobspec;
 
-    if (flux_kvs_lookup_get (f, &jobspec) < 0) {
-        flux_log_error (h, "sched.alloc lookup R");
-        goto error;
+    flux_log (util->h, LOG_INFO, "alloc_continuation: f=%p: peek=%p, count=%ld",
```

Comment:
Should this be LOG_DEBUG?  Or maybe removed?

Reply:
Oops yeah, accidentally committed some debug code, sorry!